app/assets/javascripts/browsered/web-payments/apple-pay.js

```diff
@@ -26,7 +31,8 @@ module.exports = () => {
         })
       } else {
         ga('send', 'event', 'Apple Pay', 'Error', 'Merchant ID not valid')
-        return session.abort()
+        return sendLogMessage(window.chargeId, 'ApplePayMerchantIdNotValid')
```

Comment:
I was wondering if it is worth writing this this as follows? 
As it I think it means that it might be a bit quicker - as its not waiting for `finally` block to execute?

```suggestion
        sendLogMessage(window.chargeId, 'ApplePayMerchantIdNotValid')
        return session.abort()
```

Reply:
Yeh, maybe don't need to wait for the promise to resolve, as we're doing in the other places

Reply:
Have updated